other/upgrade.php

```diff
@@ -1010,7 +1010,11 @@ function checkLogin()
 				$upcontext['user']['id'] = 1;
 				$upcontext['user']['name'] = 'Administrator';
 			}
-			$upcontext['user']['pass'] = $smcFunc['random_int'](0, 60000);
+
+            if (!is_callable('random_int'))
```

Comment:
The code is not well aligned -> because you use space and the code is using tab here.